coordinator/src/db/trades.rs

```diff
@@ -51,6 +51,57 @@ pub fn insert(
     Ok(trade.into())
 }
 
+#[derive(Debug)]
+pub struct TradeForContract {
+    pub temporary_contract_id: [u8; 32],
+    pub contract_symbol: trade::ContractSymbol,
+    pub direction: trade::Direction,
+    pub contracts: f32,
+    pub timestamp: OffsetDateTime,
+}
+
+/// Find all the trades for which we have a `temporary_contract_id`.
+///
+/// Some might not have one due to backwards compatibility.
+pub fn get_all_with_temporary_contract_id(
+    conn: &mut PgConnection,
+) -> Result<Vec<TradeForContract>> {
+    let trades: Vec<(
```

Comment:
🔧 With #1246 the coordinator will hedge expired positions to the maker, so this query should filter expired positions, as they should be hedged by the coordinator. (that should also work in retrospect for positions that have expired before the change.)